cdparanoia/fix-includes.patch

```diff
@@ -0,0 +1,7 @@
+--- cdparanoia-III-10.2.orig/utils.h
++++ cdparanoia-III-10.2/utils.h
```

Comment:
Are these also borrowed from alpine? Do we know if we need them, and what they did for alpine?

Reply:
In this case, all of these patches are needed to make it compile cleanly under modern GCC.  But they need to be documented.